csrc/scheduler/matmul.cpp

```diff
@@ -460,23 +461,29 @@ void scheduleMatmul(Fusion* fusion, const MatmulParams& params) {
       outputs[0]->isA<TensorView>(),
       "fusion's output is not an instance of TensorView class");
 
-  TORCH_INTERNAL_ASSERT(
-      mma_ops.size() == 1,
-      "scheduleMatmul supports fusion with single mma op in definition, got ",
-      mma_ops.size());
-  TORCH_INTERNAL_ASSERT(
-      mma_ops.front()->layout().has_value(),
-      "fusion mma op has undefined input layout");
+  {
+    const auto mma_ops = ir_utils::getMmaOps(fusion);
```

Comment:
I wanted to avoid having a container with MmaOps in the wider scope, but perhaps this is premature optimization. I will update this in the upcoming update of this PR.

I will also change a little bit code around handling of layout, this is needed to satisfy clang-tidy check.